### ISSUE TITLE
Unable to install on Python 3.12: module 'configparser' has no attribute 'SafeConfigParser'

### ISSUE THREAD

**Issue by carderne** [pygeos/pygeos#463]
Seems to be similar to this issue:
https://github.com/pydata/pandas-datareader/issues/969

If I run `pip install pygeos==0.14` on Python 3.12.1 I get the following error:

```
        File "<string>", line 214, in <module>
        File "/<snip>/pygeos_ec04d2bcfc0d432cacb2068bfb3c52a2/versioneer.py", line 1480, in get_version
          return get_versions()["version"]
                 ^^^^^^^^^^^^^^
        File "/<snip>/pygeos_ec04d2bcfc0d432cacb2068bfb3c52a2/versioneer.py", line 1412, in get_versions
          cfg = get_config_from_root(root)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/<snip>/pygeos_ec04d2bcfc0d432cacb2068bfb3c52a2/versioneer.py", line 342, in get_config_from_root
          parser = configparser.SafeConfigParser()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      AttributeError: module 'configparser' has no attribute 'SafeConfigParser'. Did you mean: 'RawConfigParser'?
      [end of output]
```

Seems to be solution was to replace versioneer with setuptools_scm?

**Comment by mwtoews** [comment]
It looks like you are using pygeos as a [dependency in a project](https://github.com/carderne/ccg-spider/blob/27e5b57d1f26164428ec250fb1c4c07cdb3920e1/prep/pyproject.toml#L34-L44) that uses geopandas. I'd suggest removing both shapely and pygeos from this list, as geopandas should get the correct backend dependency.

As a background note, pygeos is no longer updated, as it was merged with shapely 2.0. See the ["Migration from PyGEOS geometry backend to Shapely 2.0"](https://geopandas.org/en/stable/docs/user_guide/pygeos_to_shapely.html) doc from geopandas for more info.

**Comment by carderne** [comment]
Thanks!
